.github/workflows/node.js.yml

```diff
@@ -20,6 +20,8 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    - name: Setup unit test environment
+      run: sudo apt-get install librsvg2-bin exiftool
```

Comment:
note: you will need to mock exiftool in unit tests & remove the need for installing exiftool in the ci